solution.js

```diff
@@ -1,34 +1,110 @@
 const { catArt, someNums, someProducts, someWords } = require("./data/data.js");
 
 // sort numbers in ascending order
-const sortNumsA = () => {};
+const sortNumsA = (nums) => {
+  return nums.sort((a, b) => a - b);
+};
 
 // sort numbers in descending order
-const sortNumsD = () => {};
+const sortNumsD = (nums) => {
```

Comment:
lets do this without `.sort()`

See if you can implement a merge or quick sort during todays lab time